dev/cognitiveservices/data-plane/Language/analyzeconversations.json

```diff
@@ -2210,6 +2219,52 @@
         }
       ]
     },
+    "ChapteringParameters":{
+      "type": "object",
+      "description": "The parameters to control the chaptering.",
+      "properties": {
```

Comment:
How about a parameter for max chapter count or chapter granularity? MinChapterLength may not cover all cases of extracting "top-level" topics only, such as:
- Topic 1 (1000 chars)
- Topic 2 (1000 chars)
- Topic 3 (9000 chars)
  - Topic 3.1 (1000 chars)
  - Topic 3.2 (1000 chars)
  - ...

Reply:
How about let user specify minChapterLengthInCharacters/maxChapterLengthInCharacters per chapter level?

Reply:
That doesn't address the concern -- rather the functionality we want to support is clustering topics to extract the top K chapters where chapters may have dramatically different lengths but are semantically consistent internally.